docs/en/week12/12-3.md

```diff
@@ -255,17 +257,20 @@ class MultiHeadAttention(nn.Module):
 
         # Make sure that the embedding dimension of model is a multiple of number of heads
         assert d_model % self.num_heads == 0
-
         self.d_k = d_model // self.num_heads
-
         # These are still of dimension d_model. They will be split into number of heads
         self.W_q = nn.Linear(d_xq, d_model, bias=False)
         self.W_k = nn.Linear(d_xk, d_model, bias=False)
         self.W_v = nn.Linear(d_xv, d_model, bias=False)
-
         # Outputs of all sub-layers need to be of dimension d_model
         self.W_h = nn.Linear(d_model, d_model)
 
+```
+
+Initialization of multi-headed attention class. If a d_input is provided, this becomes cross attention. Otherwise, self-attention. The query, key, value setup is constructed as a linear transformation of the input d_model.
```

Comment:
```suggestion
Initialization of multi-headed attention class. If a `d_input` is provided, this becomes cross attention. Otherwise, self-attention. The query, key, value setup is constructed as a linear transformation of the input `d_model`.
```